packages/amplication-server/src/core/billing/billing.service.ts

```diff
@@ -35,9 +35,16 @@ export class BillingService {
     private readonly analytics: SegmentAnalyticsService,
     configService: ConfigService
   ) {
-    const stiggApiKey = configService.get(Env.BILLING_API_KEY);
-    this.stiggClient = Stigg.initialize({ apiKey: stiggApiKey });
-    this.clientHost = configService.get(Env.CLIENT_HOST);
+    const isBillingEnabled = Boolean(
+      configService.get<string>(Env.BILLING_ENABLED) === "true"
+    );
```

Comment:
@EugeneTseitlin I tried to use boolean but it didn't work. I would love to get your insight 
`this.configService.get<boolean>Env.BILLING_ENABLED);`